### DISCUSSION TITLE
Can't get accepted for the student pack.

### DISCUSSION THREAD

**Discussion by Yodakole1** [community/community#139081] [GitHub Education]
### Hi everyone,

Hi guys, I hope you're doing great!

I need some help with the student pack.

When I get rejected, I receive this in the email:

1.If your school provides an academic email, then please verify and use your academic email address. If your school does not provide an academic email address, then you must enable use of your device camera to capture your image.

2. Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

3. Please use your device camera to submit academic affiliation documents.

4. Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable.

5. You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

I added my academic email to my account as the primary email and verified it. 
My name is the same on my school ID and my GitHub account. 
I took the photos with my phone because I have a bad webcam (that's the only problem I can think of). 
I also added my billing information.

This is the third time I've been rejected, and last time I got the same email. I did everything they asked, but the same thing is still happening.

Can you guys help pls?

**Comment by alx-mp4** [comment]
Hi @Yodakole1, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
